### PR TITLE
Bug fix: JSON Bytes string validity check

### PR DESCRIPTION
**Pull request by ryley-o**
Fix bug in `bytesToJSONValue` generic handler logic used to determine if bytes value is an encoded UTF-8 string vs. raw bytes.

A false-positive "UTF-8 encoded string" was detected in our test suite in some unrelated work. Specifically, the randomly generated address `0x57e32bd396b7c3337dd6b4a672e6d99b47865e56` was tested and found to be a valid "WTF-8 encoded string", which is used internally by AssemblyScript, but not a valid "UTF-8 encoded string". Upon further investigation, the likelihood of of us attempting to upload a bytes32 not intended to be an encoded string (e.g. Merkle root) is pretty high (Based on testing a few random addresses, I would estimate anywhere from 10% to 50%, not completely sure without spending more time).

This updates our handler logic to convert from `Bytes` to string via `Bytes.toString()` method, then from string to Bytes via `string.toHexString()` method. The starting `Bytes` and the decoded/re-encoded `Bytes` will only be the same if the original `Bytes` was a valid `UTF-8` encoded string.

Note that it is possible for this to detect false-positives and improperly assume bytes are an encoded string. The likelihood of that occurring goes up with shorter bytes values being stored. When using a Bytes32, it seems negligibly likely that we would encounter a false positive, If we want to prove that we should do some more work to quantify.

## Deployment Impacts
This bug fix should be included for any work with the new Merkle minter (currently in Goerli-dev deployment) Cc @mchrupcala

### CODE REVIEW

**Comment by ryley-o** [comment]
Also Cc @yoshiwarab for visibility 👍

**Comment by ryley-o** [comment]
@Jbern16 just realized - does this need to also be considered in `handleAddManyValueGeneric` in `minter-suite-mapping.ts`?

**Review by Jbern16 — APPROVED** [review]

**Comment by Jbern16** [comment]
I think we should be good since we are using that method here - https://github.com/ArtBlocks/artblocks-subgraph/blob/main/src/minter-suite-mapping.ts#L556

**Comment by Jbern16** [comment]
Oh wait you are right - its the many handler that's not explicit about this it seems - https://github.com/ArtBlocks/artblocks-subgraph/blob/main/src/minter-suite-mapping.ts#L724-L731. That probably should updated to use this method

**Comment by ryley-o** [comment]
@Jbern16 cool cool I'll update the many handler to use this, and ping you when updated 👍

**Review by jakerockland — APPROVED** [review]

**Comment by ryley-o** [comment]
@Jbern16 updated!
This has sufficient approvals, so I'm going to merge 👍

**Comment by ryley-o** [comment]
@lyaunzbe not a big rush, but this will only merge after your approval 👍

**Review by lyaunzbe — APPROVED** [review]

**Comment by lyaunzbe** [comment]
appreciate the detailed writeup + comments here 💯

**Pull request merged**
